src/components/ExtraPages/About/About.js

```diff
@@ -1,17 +1,16 @@
 import React from "react";
 import { Container } from "react-bootstrap";
 import "../ExtraPages.css";
-import { AboutInfo } from "./Contents/AboutInfo";
 
 export default function About() {
   return (
     <Container className="about">
       <div>
-        <h1>{AboutInfo.title}</h1>
+        <h1>About UK CORDEX</h1>
         <p>
-          <a href={AboutInfo.links[1]}>
+          <a href="https://www.metoffice.gov.uk/research/approach/collaboration/ukcp/about">
```

Comment:
This link should only apply to the "UK Climate Projections (UKCP18)" bit of the text. Sorry, commit suggestions only allow for one-liners.